model_compression_toolkit/quantizers_infrastructure/common/node_quantization_dispatcher.py

```diff
@@ -32,6 +32,29 @@ def __init__(self,
         self.weight_quantizers = weight_quantizers if weight_quantizers is not None else dict()
         self.activation_quantizers = activation_quantizers if activation_quantizers is not None else list()
 
+
+    def set_weight_quantizers(self, weight_quantizers):
+        """
+        This function sets weight quantizers to existing node dispatcher
+
+        Args:
+            weight_quantizers: weight quantizers
+
+        Returns: None
+        """
+        self.weight_quantizers = weight_quantizers
```

Comment:
Add validation to the struct of the quantizers (make sure this is a dictionary of str to quantizers)